Cargo.toml

```diff
@@ -85,3 +85,4 @@ lto = true
 webpki = { git = "https://github.com/linkerd/webpki", branch = "cert-dns-names-0.22" }
 boring = { git = "https://github.com/cloudflare/boring" }
 tokio-boring = { git = "https://github.com/cloudflare/boring" }
+linkerd2-proxy-api = { git = "https://github.com/linkerd/linkerd2-proxy-api", rev = "f1598b46c179f5a981bfca7b660dc89903b18ab7" }
```

Comment:
Why wasn't a proxy API dependency present before?

Reply:
This is a dependency override.  Specifying this replaces the dependency on `linkerd2-proxy-api` everywhere in the project with one on a specific proxy-api commit.  Once we have the changes we need released in a proxy-api release, we will update the dependency to that version and remove this override.